src/injected/scan-incomplete-warning-detector.ts

```diff
@@ -11,14 +12,20 @@ export class ScanIncompleteWarningDetector {
         private permissionsStateStore: BaseStore<PermissionsStateStoreData, Promise<void>>,
     ) {}
 
-    public detectScanIncompleteWarnings = () => {
+    public detectScanIncompleteWarnings = (results: ScanResults | null) => {
         const warnings: ScanIncompleteWarningId[] = [];
+
         if (
             this.iframeDetector.hasIframes() &&
             !this.permissionsStateStore.getState().hasAllUrlAndFilePermissions
         ) {
             warnings.push('missing-required-cross-origin-permissions');
         }
+
+        if (results && results.framesSkipped) {
+            warnings.push('frame-skipped');
```

Comment:
Does it actually make sense for us to show this and the missing-permissions warning at the same time? I think it might make more sense to suppress this message if the missing-permissions case would also be triggered, since that's going to be the root cause of the issue. I think this is doubly true if we do end up adding a "file an issue" action item to the warning - we don't really want to have users report issues about the frame-tested violations if they haven't given the extension the necessary permissions.